polaris-react/src/components/AccountConnection/AccountConnection.tsx

```diff
@@ -52,43 +52,34 @@ export function AccountConnection({
     />
   ) : null;
 
-  let titleMarkup: React.ReactNode = null;
-  if (title) {
-    titleMarkup = <div>{title}</div>;
-  } else if (accountName) {
-    titleMarkup = <div>{accountName}</div>;
-  }
+  const titleMarkup = title ? title : accountName;
```

Comment:
@aveline we could bring this tiny clean up across. Unsure why it required the fragment/react component type before.